packages/fast-colors/src/colorlib/colorScale.ts

```diff
@@ -0,0 +1,203 @@
+// tslint:disable:member-ordering
+// tslint:disable:prefer-for-of
+// tslint:disable:interface-name
+
+import { contrastRatio } from "./colorConverters";
+import { ColorInterpolationSpace, interpolateByColorSpace } from "./colorInterpolation";
+import { ColorRGBA64 } from "./colorRGBA64";
+
+export interface IColorScaleStop {

```

Comment:
nit: same here, think we could do away with the "I"

Reply:
Yea good call - the reason we do this is because Interfaces and Type Aliases can be used interchangibly, but changing from one to the other can cause a breaking change if the naming convention is to use an `I` for all interface names. Not using an I to differentiate interfaces means we can change from an interface to a type alias as necessary without introducing naming changes.